src/main/java/graphql/schema/diffing/ana/EditOperationAnalyzer.java

```diff
@@ -786,11 +786,17 @@ private void fieldChanged(EditOperation editOperation) {
 
         if (fieldsContainerForField.isOfType(SchemaGraph.OBJECT)) {
             Vertex object = fieldsContainerForField;
+            if (isObjectAdded(object.getName())) {
+                return;
+            }
```

Comment:
Ran into this bug when I ran `shortcuts execution on big changes` test without the shortcut code.